app/src/tests/appConfig.test.ts

```diff
@@ -28,5 +28,21 @@ describe("api service", () => {
         /* eslint-enable */
         expect(appConfig.apiUrl).toThrow();
     });
+
+    test("uses PACKET NAMESPACE from environment", () => {
+        process.env.REACT_APP_PACKIT_NAMESPACE = "my-repo";
+        /* eslint-disable */
+        const appConfig = require("../config/appConfig").default;
+        /* eslint-enable */
+        expect(appConfig.appNamespace()).toBe("my-repo");
+    });
+
+    test("returns null when PACKET NAMESPACE is missing", () => {
```

Comment:
```suggestion
    test("returns null when PACKIT NAMESPACE is missing", () => {
```
This one too I guess!